### ISSUE TITLE
[a11y] Format requirement missing for Zip Code Edit Field.

### ISSUE THREAD

**Issue by rockycodes** [civiform/civiform#12792]
**Describe the bug**
Issue: Format requirement missing for Zip Code Edit Field.

Location: Comprehensive Sample Program Form Screen 1

**WCAG Success Criterion**
3.3.2 Labels or Instructions

**To Reproduce**
PC/JAWS
iPhone/VO

1. Navigate to Zip Code Edit Field.
 
 Format instructions not present.

**Expected behavior**
Expectation: If a Form Field has format requirements, format instructions must be provided visually and programmatically.

Remediation:
Add visual and programatic format instructions for the Zip Code Edit Field.
 *Enter 5 digits*
 *e.g. 12345*
